src/dwarf.cpp

```diff
@@ -198,22 +198,36 @@ bool has_flag_attribute(const attribute_sequence& attributes, dw::at name) {
 
 std::size_t die_hash(const die& d, const attribute_sequence& attributes) {
     ZoneScoped;
-
-    // Tag used to be a part of the fatal hash computation. Unfortunately it causes `class` and
-    // `struct` definitions to be considered different, when they should not be. As a general
-    // rule, I think _all_ symbols regardless of tag should be uniquely defined, so pulling the tag
-    // from the hash below should catch more issues without adding any false positives.
-    //
-    // The `declaration` attribute used to be a part of this hash, too. Given that a declaration
+    
+    // The `declaration` attribute used to be a part of this hash. Given that a declaration
```

Comment:
This comment about declaration is no longer relevant. I had to pull it from the `skip_die` portion of the code, but forgot about this comment. can you remove it? I am not sure what to do about the re-introduction of the declaration here as part of the `die_hash`. Thoughts?

Reply:
So, put it back to how it was a while ago?
```cpp
bool is_declaration =
        attributes.has_uint(dw::at::declaration) && attributes.uint(dw::at::declaration) == 1;
```
and include `is_declaration` in the hash?

Reply:
I have consolidated the calls into a new `has_flag_attribute` as I am using it elsewhere. So in the hash you can just include `has_flag_attribute(attributes, dw::at::declaration)`.